docs/types/value-types.rst

```diff
@@ -244,116 +244,134 @@ For a quick reference of all members of address, see :ref:`address_related`.
 
 * ``balance`` and ``transfer``
 
-It is possible to query the balance of an address using the property ``balance``
-and to send Ether (in units of wei) to a payable address using the ``transfer`` function:
+    It is possible to query the balance of an address using the property ``balance``
+    and to send Ether (in units of wei) to a payable address using the ``transfer`` function:
 
-.. code-block:: solidity
-    :force:
+    .. code-block:: solidity
+        :force:
 
-    address payable x = payable(0x123);
-    address myAddress = address(this);
-    if (x.balance < 10 && myAddress.balance >= 10) x.transfer(10);
+        address payable x = payable(0x123);
+        address myAddress = address(this);
+        if (x.balance < 10 && myAddress.balance >= 10) x.transfer(10);
 
-The ``transfer`` function fails if the balance of the current contract is not large enough
-or if the Ether transfer is rejected by the receiving account. The ``transfer`` function
-reverts on failure.
+    The ``transfer`` function fails if the balance of the current contract is not large enough
+    or if the Ether transfer is rejected by the receiving account. The ``transfer`` function
+    reverts on failure.
 
-.. note::
-    If ``x`` is a contract address, its code (more specifically: its :ref:`receive-ether-function`, if present, or otherwise its :ref:`fallback-function`, if present) will be executed together with the ``transfer`` call (this is a feature of the EVM and cannot be prevented). If that execution runs out of gas or fails in any way, the Ether transfer will be reverted and the current contract will stop with an exception.
+    .. note::
+        If ``x`` is a contract address, its code (more specifically: its :ref:`receive-ether-function`, if present, or otherwise its :ref:`fallback-function`, if present) will be executed together with the ``transfer`` call (this is a feature of the EVM and cannot be prevented). If that execution runs out of gas or fails in any way, the Ether transfer will be reverted and the current contract will stop with an exception.
+
+    .. warning::
+        ``transfer`` is deprecated and scheduled for removal.
+        Simple ether transfers can still be performed using the :ref:`call function <address_call_functions>`
+        with with an optionally provided maximum amount of gas and empty payload, i.e., ``call{value: <amount>}("")``.
+        By default this forwards all the remaining gas, subject to additional limits imposed by some EVM versions
+        (such as the `63/64th rule <https://eips.ethereum.org/EIPS/eip-150>`__ introduced by ``tangerineWhistle``).
```

Comment:
The usual link syntax uses a single underscore but you have two.

```suggestion
        (such as the `63/64th rule <https://eips.ethereum.org/EIPS/eip-150>`_ introduced by ``tangerineWhistle``).
```

It seems to work on github, but not sure how it will behave in our docs so I'd change it. Also, it could have some special meaning.

There's also one more below, in the note for `send()`.

Reply:
From what I read I understood that the double underscore would create an "anonymous" reference which could be reused in the text. I figured that would be desirable since we are repeating the link.

Reply:
From what I understood reading online, the double underscore would allow to have repeated usage of the link with the same name as it create an "anonymous" link.
I will try with the single underscore.

Reply:
Ah, so it was intentional? It's fine then. Looked like a mistake to me.

Reply:
Would be best to render the docs locally and check. Though I don't remember having any problems of this kind and I'm pretty sure we have some links in multiple places so I expect that it will be still fine.